src/lxc/lxc_execute.c

```diff
@@ -139,6 +145,12 @@ int main(int argc, char *argv[])
 	if (lxc_config_define_load(&defines, conf))
 		return 1;
 
+	if (my_args.uid)
+		conf->init_uid = my_args.uid;
```

Comment:
I convinced myself that adding these via configuration file would be useful. So, I added support in https://github.com/ptoomey3/lxc/commit/8084bd518bd7da606e21462c71af78f6c69bf31d and updated the English docs in https://github.com/ptoomey3/lxc/commit/14dcbac128bf975822d90cd986694db8e7534084. I'll leave it up to you whether you would like this set:

* Only via command line flags
* Only via the configuration file
* Both

I kind of vote both since this setting is unique to `lxc-execute`, but could also be useful to store in a configuration file for repeated use.